src/sql/workbench/contrib/executionPlan/browser/executionPlanComparisonEditorView.ts

```diff
@@ -373,12 +415,13 @@ export class ExecutionPlanComparisonEditorView {
 			this._bottomPlanDropdown.select(preSelectIndex);
 			this._propertiesView.setBottomElement(executionPlanGraphs[0].root);
 			this._addExecutionPlanAction.enabled = false;
+			this._searchNodeActionForAddedPlan.enabled = true;
 		}
 		this.refreshSplitView();
 	}
 
 	private async getSkeletonNodes(): Promise<void> {
```

Comment:
FYI, right now you have this async but it's not actually doing any async work. You call `withProgress` but then don't await it - did you mean to do that? Currently it kicks off the progress task but then immediately returns (while the async progress task is running in the background)